ui/app/src/context/Authorization.tsx

```diff
@@ -56,6 +57,20 @@ export function useAuthorizationContext(): AuthorizationContext {
   return ctx;
 }
 
+export function useDashboardCreateAllowedProjects(): ProjectResource[] {
+  const { enabled, userPermissions } = useAuthorizationContext();
+  const { data } = useProjectList();
+  if (!enabled) {
+    return data ?? [];
+  }
+
+  return (data ?? []).filter(
+    (project) =>
+      permissionListHasPermission(userPermissions[GlobalProject] ?? [], 'create', 'Dashboard') ||
+      permissionListHasPermission(userPermissions[project.metadata.name] ?? [], 'create', 'Dashboard')
```

Comment:
what would happen if the permissions are deactivated ?

Reply:
The line just before 😄 
```
if (!enabled) {
    return data ?? [];
  }
```